openshift/template.yaml

```diff
@@ -181,6 +181,9 @@ parameters:
 - name: ENABLE_DATA_COLLECTION
   value: "false"
   required: false
+- name: INSTALLER_CAPACITY
```

Comment:
let's also add some kind of feature-flag. it can either be a separate parameter, or you can assign some value of ``INSTALLER_CAPACITY`` that implies "do no caching". I tend to the first option, but it's not critical

Reply:
Maybe `INSTALLER_CACHE_LIMIT` would clarify that this is imposing a limit on cache size, and that leaving it unset means there's no limit.